components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/impl/SCIMUserManager.java

```diff
@@ -2868,14 +2868,16 @@ private Group addDomainToUserMembers(Group group, String userStoreDomain) throws
         // If the limit is zero, all the users needs to be returned after verifying the offset.
         if (limit <= 0) {
             if (offset == 1) {
+
                 // This is to support backward compatibility.
                 return users;
             } else {
                 return Arrays.copyOfRange(users, offset - 1, users.length);
             }
         } else {
-            // If users.length > limit + offset, then return only the users bounded by the offset and the limit.
             if (users.length > limit + offset) {
+
+                // If users.length > limit + offset, then return only the users bounded by the offset and the limit.
```

Comment:
since we know the condition here, we can remove it from the comment.

Reply:
Fixed: https://github.com/wso2-extensions/identity-inbound-provisioning-scim2/pull/216/commits/4b1205253880b74d0ead887c659eb3086e080187